docker.Makefile

```diff
@@ -105,7 +105,7 @@ shellcheck: build_shell_validate_image ## run shellcheck validation
 	docker run -ti --rm $(ENVVARS) $(MOUNTS) $(VALIDATE_IMAGE_NAME) make shellcheck
 
 .PHONY: test-e2e ## run e2e tests
-test-e2e: test-e2e-non-experimental test-e2e-experimental test-e2e-containerized
```

Comment:
They depended on the old init flow which we didn't port over to the updated model.  It's possible we might be able to come up with a new e2e test flow for this, but given other bugs we've been hitting on the packaging side, I don't think those tests belong in this tree, since it's not actually helping us find the interesting bugs.